ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.component.js

```diff
@@ -18,10 +18,10 @@ export const imageHash = {
   [TRANSACTION_SUBMITTED_EVENT]: 'fa-arrow-up',
   [TRANSACTION_RESUBMITTED_EVENT]: 'fa-retweet',
   [TRANSACTION_CONFIRMED_EVENT]: 'fa-check',
-  [TRANSACTION_DROPPED_EVENT]: 'fa-times',
+  [TRANSACTION_DROPPED_EVENT]: `${ICON_NAMES.CLOSE}`,
   [TRANSACTION_ERRORED_EVENT]: 'fa-exclamation',
-  [TRANSACTION_CANCEL_ATTEMPTED_EVENT]: 'fa-times',
-  [TRANSACTION_CANCEL_SUCCESS_EVENT]: 'fa-times',
+  [TRANSACTION_CANCEL_ATTEMPTED_EVENT]: `${ICON_NAMES.CLOSE}`,
+  [TRANSACTION_CANCEL_SUCCESS_EVENT]: `${ICON_NAMES.CLOSE}`,
```

Comment:
Check how this components works. It adds the font-awesome class to the component `className` these updates will break the component because it's passing the imageHash to the class name not to the Icon component

<img width="804" alt="Screenshot 2023-02-06 at 3 23 21 PM" src="https://user-images.githubusercontent.com/8112138/217109872-759f7cfa-4886-4941-bc4d-11be8cdbd631.png">